app/forms/contact/create_chair_of_governors_contact_form.rb

```diff
@@ -0,0 +1,24 @@
+class Contact::CreateChairOfGovernorsContactForm < Contact::CreateProjectContactForm
+  attribute :category, default: "school_or_academy"
+
+  def save
```

Comment:
Depending on your taste, we could loose duplicating the `save` method and overide the getters, the subclasses would then be:

```ruby
  def category
    "school_or_academy"
  end

  def title
    "Headteacher"
  end

  def organisation_name
    nil
  end
```

When `organisation_name` is nil we default to the establishment name, it could be:

```ruby
  def organisation_name
    @project.establishment&.name
  end
```

if we want to be more obvious.

Reply:
This is great, I had over-egged it a bit